src/targets/gpu/include/migraphx/gpu/mlir.hpp

```diff
@@ -50,6 +50,8 @@ struct MIGRAPHX_GPU_EXPORT mlir_code_object
     std::vector<value> prefill_values   = {};
 };
 
+MIGRAPHX_GPU_EXPORT bool is_reduce(const instruction& ins);
```

Comment:
I dont see this used outside of mlir.cpp. I think it can be removed from the header.

Reply:
It is beign used on both `fuse_mlir.cpp` and `mlir.cpp`